src/SDKs/_metadata/datalake-analytics_data-plane.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/datalake-analytics/data-plane/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=A:\azure-sdk-for-net\src\SDKs
```

Comment:
Is anything being generated for dataplane in this PR?
If not, please revert this file.

Reply:
Thanks, this is a dataplane related PR. The swagger spec file can be found [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/datalake-analytics/data-plane/Microsoft.DataLakeAnalytics/stable/2016-11-01/catalog.json).

Reply:
I see, Shouldn't be a blocker.